README.md

```diff
@@ -1 +1,72 @@
-# My-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My-Portfolio
+
+> One paragraph statement about the project.
```

Comment:
Please add a better description of your project, try to think about what you did, what kind of technologies you used, what is the main scope, and so on.

- [ ] Improve description for your project.